src/zig.ts

```diff
@@ -0,0 +1,2746 @@
+    insertValue: "-ffp-model={cursor}",
+    args: {
+      name: "value",
+    },
+  },
+  {
+    name: "-ffreestanding",
+    description:
+      "Assert that the compilation takes place in a freestanding environment",
+  },
+  {
+    name: "-ffuchsia-api-level",
+    description: "Set Fuchsia API level",
+    insertValue: "-ffuchsia-api-level={cursor}",
+    args: {
+      name: "value",
+    },
+  },
+  {
+    name: "-ffunction-sections",
+    description: "Place each function in its own section",
+  },
+  {
+    name: "-fglobal-isel",
+    description: "Enables the global instruction selector",
+  },
+  {
+    name: "-fgnu-keywords",
+    description: "Allow GNU-extension keywords regardless of language standard",
+  },
+  {
+    name: "-fgnu-runtime",
+    description:
+      "Generate output compatible with the standard GNU Objective-C runtime",
+  },
+  {
+    name: "-fgnu89-inline",
+    description: "Use the gnu89 inline semantics",
+  },
+  {
+    name: "-fgnuc-version",
+    description:
+      "Sets various macros to claim compatibility with the given GCC version (default is 4.2.1)",
+    insertValue: "-fgnuc-version={cursor}",
+    args: {
+      name: "value",
+    },
+  },
+  {
+    name: "-fgpu-allow-device-init",
+    description: "Allow device side init function in HIP (experimental)",
+  },
+  {
+    name: "-fgpu-default-stream",
+    description:
+      "Specify default stream. The default value is 'legacy'. (HIP only)",
+    insertValue: "-fgpu-default-stream={cursor}",
+    args: {
+      name: "value",
+    },
+  },
+  {
+    name: "-fgpu-defer-diag", //291
+    description: "Defer host/device related diagnostic messages for CUDA/HIP",
+  },
+  {
+    name: "-fgpu-flush-denormals-to-zero",
+    description:
+      "Flush denormal floating point values to zero in CUDA/HIP device mode",
+  },
+  {
+    name: "-fgpu-rdc",
+    description:
+      "Generate relocatable device code, also known as separate compilation mode",
+  },
+  {
+    name: "-fgpu-sanitize",
+    description: "Enable sanitizer for AMDGPU target",
+  },
+  {
+    name: "-fhip-fp32-correctly-rounded-divide-sqrt",
+    description:
+      "Specify that single precision floating-point divide and sqrt used in the program source are correctly rounded (HIP device compilation only)",
+  },
+  {
+    name: "-fhip-kernel-arg-name",
+    description: "Specify that kernel argument names are preserved (HIP only)",
+  },
+  {
+    name: "-fhip-new-launch-api",
+    description: "Use new kernel launching API for HIP",
+  },
+  {
+    name: "-fignore-exceptions",
+    description: "Enable support for ignoring exception handling constructs",
+  },
+  {
+    name: "-fimplicit-module-maps",
+    description: "Implicitly search the file system for module map files",
+  },
+  {
+    name: "-finline-functions",
+    description: "Inline suitable functions",
+  },
+];
+
+const completionSpec: Fig.Spec = {
+  name: "zig",
```

Comment:
This is used to address all the flags that are "not in the standard style".
```suggestion
  name: "zig",
  parserDirectives: {
    flagsArePosixNoncompliant: true,
  },
```